checkQC/qc_checkers/cluster_pf.py

```diff
@@ -0,0 +1,41 @@
+from checkQC.handlers.qc_handler import QCErrorFatal, QCErrorWarning
+
+
+def cluster_pf(
+    self,
+    error_threshold,
+    warning_threshold
+):
+    assert self.sequencing_metrics
+    assert (
+        (error_threshold == "unknown" or warning_threshold == "unknown")
+        or error_threshold < warning_threshold
+    )
+
+    def _qualify_error(total_cluster_pf, lane):
+        data = {
+            "lane": lane,
+            "total_cluster_pf": total_cluster_pf,
```

Comment:
```suggestion
            "total_cluster_pf": total_cluster_pf / 10**6,
```
Thresholds are specified in millions so we need to adjust for that.

Reply:
Good catch! In general it's best practice to avoid converting data from one type to the other unnecessarily (e.g. from int to float), so I opted to keep everything as integers instead, except when the data is presented to the user.

Reply:
Although now I realize it might be a float already in the config :thinking: In general I don't like to mix, so I'm still leaning towards converting everything to ints, to follow the data we get from the interop files.